### PR TITLE
MM-13952: Small style and text changes.

### PR DESCRIPTION
**Pull request by mkraft**
#### Summary
Small style and text changes.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-13952

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Added or updated unit tests (required for all new features)
- [ ] Needs to be implemented in mobile (link to PR or User Story)
- [ ] Has server changes (please link)
- [ ] Has redux changes (please link)
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, posting, etc.)

### CODE REVIEW

File: components/admin_console/group_settings/group_details/group_users_row.jsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default class GroupUsersRow extends React.PureComponent {
                                 defaultMessage='Email:'
                             />
                         </span>
-                        <span className='email'>{this.props.email}</span>
+                        <span className='email'>&nbsp;{this.props.email}</span>
```
Comment (jespino) [inline components/admin_console/group_settings/group_details/group_users_row.jsx]: Why this instead of styles? I don't love the &nbsp; approach.

**Comment by mkraft** [comment]
@michaelgamble You can login to the above spinmint as sysadmin and review the style changes.

**Comment by mkraft** [comment]
@michaelgamble There was a considerable amount of setup involved in this spinmint so if you could please review the changes in the next day - before the spinmint is automatically destroyed by the 7-day limit - that would be appreciated.

**Comment by mkraft** [comment]
@michaelgamble I have setup a new spinmint for you because the old one was destroyed. This PR is awaiting your review.

**Pull request merged**